src/common/paths/query_ast.yml

```
@@ -13,20 +38,58 @@ post:
         enum:
           - application/json
   requestBody:
-    description: Analyzed Flux query to generate abstract syntax tree.
+    description: Analyzed Flux query to generate an AST.
     content:
       application/json:
         schema:
           $ref: "../schemas/LanguageRequest.yml"
+  x-codeSamples:
+    - lang: Shell
+      label: cURL
+      source: |
+        curl --request POST 'INFLUX_URL/api/v2/query/ast' \
+        --header 'Content-Type: application/json' \
+        --header 'Accept: application/json \
+        --header 'Authorization: Token INFLUX_TOKEN' \
+        --data 'from(bucket: "example-bucket")
+                  |> range(start: -5m)
+                  |> filter(fn: (r) => r._measurement == "example-measurement")'
   responses:
     "200":
-      description: Abstract syntax tree of the flux query.
+      description: |
+        Successful request.
+        AST of the Flux query.
       content:
         application/json:
           schema:
             $ref: "../schemas/ASTResponse.yml"
+          examples:
+            $ref: "../schemas/ASTSuccessResponse.yml"
+    "400":
+      description: |
+        Bad request.
+        InfluxDB is unable to parse the request.
+        The response body contains detail about the problem.
+      headers:
+        X-Platform-Error-Code:
+          description: |
+            The reason for the error.
+          schema:
+            type: string
+            example: "invalid"
+      content:
+        application/json:
+          schema:
+            $ref: "../schemas/Error.yml"
+          examples:
+            invalidASTValue:
+              summary: Invalid AST
+              value: {
+                  "code": "invalid",
+                  "message": "invalid AST: loc 1:6-1:19: missing property key"
+                }
     default:
-      description: Any response other than 200 is an internal server error
+      description: Any response other than 200 is an internal server error.
```

```suggestion
      description: Internal server error.
```

"Any response other than 200" isn't true; `400` isn't. I know this occurs throughout the specs. I've recently tried to refactor the error responses some for reuse. We should probably try a mass find/replace as long as the error responses are valid.

---

@sunbryely-influxdata For example: https://github.com/influxdata/openapi/commit/b4e0dc810e43173fa287595549a4f684e8b331c2#diff-a95672a6cf777e411fc7b612dae5b524caf34c3f47cd5d90f017eadbc8620447R17-R37